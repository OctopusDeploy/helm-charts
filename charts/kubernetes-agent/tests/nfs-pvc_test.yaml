suite: "persistence"
templates:
- templates/nfs-pvc.yaml
tests:
- it: "should match snapshot"
  asserts:
  - matchSnapshot: {}
- it: "is not created when NFS is disabled"
  set:
    persistence:
      nfs:
        enabled: false
  asserts:
  - hasDocuments:
      count: 0

- it: "is created when NFS is enabled"
  set:
    persistence:
      nfs:
        enabled: true
  asserts:
  - hasDocuments:
      count: 1
