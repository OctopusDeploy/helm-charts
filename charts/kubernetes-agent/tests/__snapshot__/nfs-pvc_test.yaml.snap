should match snapshot:
  1: |
    apiVersion: v1
    kind: PersistentVolumeClaim
    metadata:
      name: octopus-agent-nfs-release-name-pvc-10gi
    spec:
      accessModes:
        - ReadWriteMany
      resources:
        requests:
          storage: 10Gi
      storageClassName: octopus-agent-nfs-RELEASE-NAME-csi
      volumeName: octopus-agent-nfs-release-name-pv-10gi
